flutter_common/lib/src/utils/env_loader.dart

```diff
@@ -11,23 +11,30 @@ String envFilePath() {
   return env.isNotEmpty ? '$envsAssetPath/$env' : '$envsAssetPath/.env';
 }
 
+String? getEnv(String name) {
+  return dotenv.env[name] ?? (bool.hasEnvironment(name) ? String.fromEnvironment(name) : null);
+}
+
 Future<void> loadEnv() async {
   await dotenv.load(fileName: envFilePath());
-  final supabaseUrl = dotenv.env['STUDYU_SUPABASE_URL']!;
-  final supabaseAnonKey = dotenv.env['STUDYU_SUPABASE_PUBLIC_ANON_KEY']!;
+  final supabaseUrl = getEnv('STUDYU_SUPABASE_URL');
+  final supabaseAnonKey = getEnv('STUDYU_SUPABASE_PUBLIC_ANON_KEY');
+  final envAppUrl = getEnv('STUDYU_APP_URL');
+  final envDesignerUrl = getEnv('STUDYU_DESIGNER_URL');
+  final envProjectGeneratorUrl = getEnv('STUDYU_PROJECT_GENERATOR_URL');
   await Supabase.initialize(
-    url: supabaseUrl,
-    anonKey: supabaseAnonKey,
+    url: supabaseUrl!,
+    anonKey: supabaseAnonKey!,
     authCallbackUrlHostname: kIsWeb ? null : 'designer.studyu.health', // optional
     debug: true,
     // optional
   );
   env.setEnv(
-    dotenv.env['STUDYU_SUPABASE_URL']!,
-    dotenv.env['STUDYU_SUPABASE_PUBLIC_ANON_KEY']!,
-    envAppUrl: dotenv.env['STUDYU_APP_URL'],
-    envDesignerUrl: dotenv.env['STUDYU_DESIGNER_URL'],
-    envProjectGeneratorUrl: dotenv.env['STUDYU_PROJECT_GENERATOR_URL'],
+    supabaseUrl,
+    supabaseAnonKey,
```

Comment:
Why is it possible to pass an optional String into `setEnv` here when it needs a non-optional String?👀

Reply:
Dart runtime would already have crashed before if these weren't available:

```suggestion
    url: supabaseUrl!,
    anonKey: supabaseAnonKey!,
```